posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 21 May 2020"
+author: Val Grimm
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+---
+
+Hello everyone! 
+
+This blog post summarizes the thematically-related meetings of June 4, June 25, and July 9th. 
+
+## Big picture themes
+Define the role of domain working groups and community groups in the Rust Project; Rust governance taxonomy
+
+You can find the <br>
+[detailed minutes June 4](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.04.md)  <br>
+[detailed minutes June 25](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.25.md)  <br>
+[detailed minutes July 9](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.07.09.md)  <br>
+on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: <br>
+
+* There is a need for more clarity about why and how groups are formed, 
+    * what group categories mean, 
+    * how they relate to each other, 
+    * how they relate to the Rust Project
+    * and their level of decision-making power
+* Rust repository structure and permissions should ideally represent status of groups in the project
+* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
+Rust materials. Instead, it is likely that, to avoid some community groups being perceived to be "more official", 
```

Comment:
Hmm, I'm not sure I'd say it's *likely* that the rust org would like to Mozilla's community portal. I don't know that I think linking to Mozilla's community portal makes sense, myself, given that the Rust community and Mozilla community can be somewhat distinct (if heavily overlapping). I guess I would say something like "One possibility might be to link to other community portals, such as Mozilla's community portal, where folks can list those groups."

Reply:
Yeah I find the mozilla mention here to be out of place. It's fine as an example, but we prob shouldn't be linking there

Reply:
I guess we _could_ use that portal as a hosting space. But it seems unlikely.

Reply:
Have adjusted post accordingly. Also revised based on Zulip-based input from XAMPPRocky about where to put minutes link.